.github/workflows/bump-go.yaml

```diff
@@ -0,0 +1,181 @@
+# Copyright 2020 The Knative Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Before running this workflow make sure that GitHub Actions and test-infra
+# support the target Go version.
+
+name: Bump Go
+
+on:
+  workflow_dispatch:
+    inputs:
+      target:
+        description: 'Go target version'
+        required: true
+        default: '1.15'
+      branch:
+        description: 'Branch name'
+        required: true
+        default: 'master'
```

Comment:
I'd drop this here and below.  I think it's unlikely that we'll bump Go on release branches and this makes things complicated for repos that are using `main`.